components/event/EventTabs.js

```diff
@@ -0,0 +1,63 @@
+export function EventTabs({ tabs, eventType, setEventType }) {
+  const classNames = (...classes) => classes.filter(Boolean).join(" ");
+  const changeTab = (e, value) => {
+    setEventType(value);
```

Comment:
I think you might need to `e.preventDefault()` here, we had a similar issue on the `UserTabs`

Have a look and see if you can keep it similar to those tabs `components/user/UserTabs.js`